book/src/notes.md

```diff
@@ -1,87 +1,67 @@
-# Note
+# Notes
 
-`Note` is an immutable object that represents a unit of value of a certain application. Each note has an [owner](users.md), a [application](application.md) type, and a value:
+A **note** is an immutable object that represents a part of the application state. Each note belongs to a certain application (that defines the note's type), and can store some data.
 
-```rust
-pub struct Note<CP: CircuitParameters> {
-	pub owner: User<CP>,
-	pub app: App<CP>,
-	pub value: u64,
-...
-}
-```
+As notes are immutable, notes cannot be modified, only spent or created. "Modifying" a note would be done by spending the current note and creating the same, but modified, note.
```

Comment:
It makes more sense to me to talk about spending the current note and creating a note or multiple notes of the same type (maintaining the balance)

Reply:
Yeah, not the best wording